samples/sensor/ams_as5600/README.rst

```diff
@@ -0,0 +1,37 @@
+.. _ams_as5600:
+
+AMS AS5600 Angle position sensor sample
```

Comment:
We are trying to stop propagating driver-specific sensor samples. Will you please look at making this sample driver-agnostic, similar to `samples/sensor/accel_polling` or `samples/sensor/magn_polling`?

Reply:
@MaureenHelm looking to these samples I see in this case, the best option here is to remove this sample right? The shell usage would offer the same value and will not introduce more pollution to the sensor samples folder.

Other point is to remove this sample, and create a polling angle sensor  sample in a separate PR.

What do you think?

Reply:
@MaureenHelm , just following up

Reply:
Either option is fine